src/test/scala/uk/gov/nationalarchives/notifications/JenkinsBackupIntegrationSpec.scala

```diff
@@ -0,0 +1,38 @@
+package uk.gov.nationalarchives.notifications
+import org.scalatest.prop.TableFor3
```

Comment:
Maybe add a line break between package and import statements, for consistency?

Reply:
Fixed